app/views/records/_records_list.html.erb

```diff
@@ -4,6 +4,10 @@
   <% result_values = get_result_values(patients, [measure], measure.population_ids.keys(), 'patient_id') %>
 <% end %>
 <h1><%= measure ? measure.display_name : 'All' %> Patients</h1>
+<% if @vendor %>
+  <% patients= @vendor.patients %>
+  <% bundle= Bundle.all.to_a.first #temp for test%>
```

Comment:
Why cast to an array and get the first rather than just get the first out of the Bundle collection? Also is the comment on this still applicable, and if it is... does that mean that this line is no longer needed?

Reply:
Good catch! Yes- this line should be removed. PR now updated as such.